src/gluonts/evaluation/_base.py

```diff
@@ -292,6 +304,26 @@ def get_metrics_per_ts(
             "OWA": np.nan,  # by default not calculated
         }
 
+        if self.custom_eval_fn is not None:
+            for k, (eval_fn, _, fcst_type) in self.custom_eval_fn.items():
+                try:
+                    val = {
+                        k: eval_fn(
+                            pred_target,
+                            eval(
+                                fcst_type + "_fcst",
+                                {
+                                    "median_fcst": median_fcst,
+                                    "mean_fcst": mean_fcst,
+                                },
+                            ),
```

Comment:
Since it's just two values, maybe it is better to use an `if` here. We can then rework this whenever needed